### DISCUSSION TITLE
Food product emission breakdown

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#25] [Q&A] [answered]
Hi everyone, I am calculating the emissions for food products using spend data. 
I wonder if the emission factors can be broken down into stages (farm, manufacturing, packaging, logistics, etc.)?
Besides, the emission factors only account for cradle-to-gate emissions, while land use change emissions are also considered important. Are there any ways to quantify the land use change emission of food products (e.g., beef, coffee, chocolate, etc.) by using spend data?
Thank you!

**Comment by WesIngwersen** [comment, marked as answer]
Theoretically it would be possible to break the factors down into select stages. We don't offer this at the current time. 

We have procedures for contribution analysis, which can "decompose" the supply chain for a given factor. This is how we for instance derived the information to create Figure 15 in the original [report](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324). Here is a clip from that figure for the Food and Beverages sector

<img width="230" alt="image" src="https://user-images.githubusercontent.com/5831306/184365782-c7ee25ae-ed7d-41e4-9f8c-a3611dc6ccc4.png">

The commodities in the figure could used for this classification - for instance `111CA` is Farms so this could represent the farm stage, `484` is transportation so this could present transport from farm to manufacture although doing so wouldn't be perfect because some of that transportation would come from anywhere in the supply chain network.

**Comment by WesIngwersen** [comment]
@arby1005 I consider the second question to be a separate topic on land-use change emissions.  I will move it into another discussion.
